forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -328,14 +329,15 @@ def form_design_update(data, form_id):
 
     @classmethod
     @user_context
-    def create_default_process(cls, process_name, **kwargs):
+    def create_default_process(cls, process_name, status="DRAFT", **kwargs):
         """Create process with default workflow."""
         user: UserContext = kwargs["user"]
         process_dict = {
             "name": process_name,
             "process_key": process_name,
             "parent_process_key": process_name,
             "process_type": "BPMN",
```

Comment:
It would be better to move all these strings process_type, status etc. to be moved to an enum or a constant file